static/styles/lib/loggedin.scss

```diff
@@ -814,3 +814,66 @@ footer {
 .school-logo {
 	max-height: 60px;
 }
+
+.content-wrapper .cloud-logo {
+	display: inline-block;
+	vertical-align: top;
+	width: 50px;
+	height: 30px;
+	margin: -5px 0.25rem 0 0;
+	background: $logoBackground;
+	background-size: contain;
+
+	@include media-breakpoint-down(md) {
+		width: 100%;
+		margin-right: 0;
+	}
+
+	@include media-breakpoint-down(xs) {
+		width: 45px;
+		margin-right: 0;
+	}
+}
+
+@media (max-width: 575px) {
+	.topbar.navbar {
+		height: 56px;
+	}
+
+
+
+	.sc-branding {
+		width: fit-content;
+	}
+
+  .breadcrumb,
+  .btn-fullscreen,
+  .qr-code-action,
+  .help-drodown {
+    display: none !important;
+  }
+
+	.content-wrapper .cloud-logo {
+		width: 9.375em;  /* (150px / 16px) */
```

Comment:
is the calculation important for something? if it is, then put the origin values into variables and calculate the value here based on them. if not: remove the comment